src/olympia/devhub/views.py

```diff
@@ -1469,14 +1469,26 @@ def _submit_upload(request, addon, channel, next_view, wizard=False):
         if existing_properties
         else []
     )
+    submit_notification_warning = get_config('submit_notification_warning')
+    if not submit_notification_warning and addon:
+        # If we're not showing the generic submit notification warning, show
+        # one specific to pre review if the developer would be affected because
+        # of its promoted group.
+        promoted_group = addon.promoted_group(currently_approved=False)
+        if (channel == amo.CHANNEL_LISTED and promoted_group.listed_pre_review) or (
+            channel == amo.CHANNEL_UNLISTED and promoted_group.unlisted_pre_review
+        ):
+            submit_notification_warning = get_config(
+                'submit_notification_warning_pre_review'
+            )
```

Comment:
The wording of this config is going to have to be a little tricky if it has to cover both groups that have `listed_pre_review` but not `unlisted_pre_review`, and groups that have both.  If we do want to have this for all groups (and I can see why that's a good idea rather than special casing it to Notable) then perhaps we want two new configs so the `listed_pre_review` only group can mention listed versions specifically?

Reply:
I figured since we are only showing it at the upload step, then it was ok to use a generic "pre_review" key that didn't specify the channel: we know the channel already from the previous step, so we're only showing the banner where relevant.